pkgs/tools/filesystems/moosefs/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchzip
+, fetchFromGitHub
+, makeWrapper
+, python
+, fuse
+, pkgconfig
+, libpcap
+, file
+, zlib 
+}:
+
+stdenv.mkDerivation rec {
+  name = "moosefs-${version}";
+  version = "3.0.103";
+
+  src = fetchFromGitHub {
+    owner = "moosefs";
+    repo = "moosefs";
+    rev = "v${version}";
+    sha256 = "0pqralv57ci4zwd75hz4pxmd4l9d4nib2mcsvrb6jndxqkaqcvns";
+  };
+
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
+
+  buildInputs =
+    [ fuse libpcap zlib   
+    ];
```

Comment:
The formatting of these lines is kind of strange. How about this?

```nix
buildInputs = [ fuse libpacp zlib ];
```

Reply:
Fixed in the latest commit!